src/components/promo/promo.jsx

```diff
@@ -0,0 +1,29 @@
+var React = require('react');
+
+module.exports = React.createClass({
+  propTypes: {
+    logoAlt: React.PropTypes.string.isRequired,
+    logoSrc: React.PropTypes.string.isRequired,
+    head: React.PropTypes.string.isRequired,
+    subhead: React.PropTypes.string.isRequired,
+    ctaText: React.PropTypes.string.isRequired,
+    ctaLink: React.PropTypes.string.isRequired
+  },
+  handleBtnClick: function () {
+    window.location.href = this.props.ctaLink;
+  },
+  render: function () {
+    return (
+      <div className="promo">
+        <div className="inner">
+          <img alt={this.props.logoAlt}
+               src={this.props.logoSrc}
+               width="100" />
+          <h3>{this.props.head}</h3>
+          <h4>{this.props.subhead}</h4>
+          <button onClick={this.handleBtnClick}>{this.props.ctaText}</button>
```

Comment:
Yea... I got really confused about the [`.btn`](https://github.com/mozilla/webmaker-desktop/blob/develop/src/components/button/button.less) and [`button`] stylings (https://github.com/mozilla/webmaker-desktop/blob/develop/src/pages/splash/splash.less#L27-L48)(<--- there's no `<button>` in `splash.jsx` either :no_mouth: ) in this repo.  Really don't wanna override all the existing stylings... but maybe that's the way it should be?